src/components/SearchMap.js

```diff
@@ -56,20 +65,94 @@ const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites, navigate})
+        const featureIcons = _.map(features, feature => {
+            return (
+                <Icon
+                    key={feature}
+                    reverse
+                    size={15}
+                    name={featureIconDetails[feature].name}
+                    type={featureIconDetails[feature].type}
+                />
+            )
+        });
+
+        const facilityIcons = _.map(facilities, facility => {
+            return (
+                <Icon
+                    key={facility}
+                    reverse
+                    size={15}
+                    name={facilityIconDetails[facility].name}
+                    type={facilityIconDetails[facility].type}
+                />
+            );
+        });
+
+        return _.concat(featureIcons, facilityIcons);
+    };
+
+    const renderSelectedSitePreview = () => {
+        if (selectedSite && !_.isEmpty(selectedSite)) {
+            const {id, title, description, nearestTown, accessibility, siteImageData, features, facilities} = selectedSite;
+            const {sitePreviewContainerStyle, touchableMainContainerStyle, mainInnerContainerStyle, topRowInfoStyle, topRowText, titleRowStyle, bottomRowInfoStyle, bottomRowText, closeIconStyle, IconContainer} = styles;
+
+            return (
+                <ImageBackground
+                    pointerEvents="none"
+                    source={siteImageData ? {uri: `data:image/png;base64,${siteImageData}`} : require('../../assets/starTent.jpg')}
+                    style={sitePreviewContainerStyle}
+                >
+                    <TouchableOpacity style={touchableMainContainerStyle} onPress={() => getSiteDetail({selectedSite: selectedSite, navigate})}>
+                        <View style={mainInnerContainerStyle}>
+                            <View style={topRowInfoStyle}>
+                                <View>
+
+                                </View>
+                                <Icon style={closeIconStyle} type='ionicon' name='md-close-circle' size={40}
+                                      color={'white'}
+                                      onPress={() => getSiteDetail({selectedSite: null})}/>
+                            </View>
+
+                            <View style={bottomRowInfoStyle}>
+                                <View style={titleRowStyle}>
+                                    <Text h4 style={bottomRowText}>{title}</Text>
+                                    <Text style={bottomRowText}> - </Text>
+                                    <Text style={bottomRowText}>{accessibility_options[accessibility]}</Text>
+                                </View>
+
+                                <View style={IconContainer}>
+                                    {renderIcons({features, facilities})}
+                                </View>
+                            </View>
+                        </View>
+                    </TouchableOpacity>
+
+                </ImageBackground>
```

Comment:
Above is the site preview for the MapView. It's all sitting in an `ImageBackground` (Thanks for showing me that component @amgoodman5 !), and then wrapped by a `TouchableOpacity` to essentially make the entire container a clickable object. Then we have a top row and bottom row. Top just holds the X button for now, and the Bottom has the site info.